sway/sway.5.scd

```diff
@@ -294,11 +294,17 @@ runtime.
 	the _--no-warn_ flag.
 
 	Mouse buttons can either be specified in the form _button[1-9]_ or by using
-	the name of the event code (ex _BTN_LEFT_ or _BTN_RIGHT_). For the former
+	the name of the event code (ex _BTN\_LEFT_ or _BTN\_RIGHT_). For the former
```

Comment:
@ddevault With scdoc 1.8, is it intentional that the underscores have to be escaped here to underline the whole word? I'm not sure due to the other underscores.

Reply:
Yes

Reply:
1.8.1 rather